isis/src/clipper/apps/jitterfit/jitterfit.xml

```diff
@@ -52,7 +104,7 @@
           This input cube is the check line cube. It contains a set of lines that were 
           repetedly read from the sensor array intersperesed with the readout of the main
           image data.
-          EXPAND
+          ...EXPAND...
```

Comment:
I agree with the expand comment you put in here: Are there any assumptions made or formatting constraints on the check line cube? What does this program expect of the check line cube?
I even think something like an example of a specific case might help like: "lines 1, 30, and 60 are read out n times each and placed into the check line cube" (in what order? 1 1 1 1 30 30 30 30 60 60 60 60 or 1 30 60 1 30 60, etc) 

Or,  don't go into this detail here, and say "it expects the format that `eis2isis` outputs," and we can add this documentation over there.

Reply:
The read order is going to be instrument dependent, and may change depending on research done on the first few real results. I don't want to put hard rules in here because we don't have them and they are going to change. I'm going to leave this as it just needs to be expanded on when things quit changing.

Reply:
I think you addressed this with your added comment about how this program was developed to support EIS.